http.kafka.cqrs/zilla.json

```diff
@@ -0,0 +1,201 @@
+            {
+                "host": "0.0.0.0",
+                "port": 8080
+            },
+            "exit": "tls0"
+        },
+        "tcp1":
+        {
+            "type" : "tcp",
+            "kind": "server",
+            "options":
+            {
+                "host": "0.0.0.0",
+                "port": 8080
+            },
+            "exit": "http0"
+        },
+        "tls0":
+        {
+            "type" : "tls",
+            "kind": "server",
+            "vault": "server",
+            "options":
+            {
+                "keys": [ "localhost" ],
+                "sni": [ "localhost" ],
+                "alpn": [ "http/1.1", "h2" ]
+            },
+            "exit": "http0"
+        },
+        "http0":
+        {
+            "type" : "http",
+            "kind": "server",
+            "routes":
+            [
+                {
+                    "exit": "http_kafka0",
+                    "when":
+                    [
+                        {
+                            "headers":
+                            {
+                                ":scheme": "http",
+                                ":authority": "localhost:8080"
+                            }
+                        },
+                        {
+                            "headers":
+                            {
+                                ":scheme": "https",
+                                ":authority": "localhost:9090"
+                            }
+                        }
+                    ]
+                },{
+                    "exit": "sse0",
+                    "when":
+                    [
+                        {
+                            "headers":
+                            {
+                                ":path": "/tasks"
+                            }
+                        }
+                    ]
+                }
+            ]
+        },
+        "http_kafka0":
+        {
+            "type" : "http-kafka",
+            "kind": "proxy",
+            "routes":
+            [
+                {
+                    "exit": "kafka0",
+                    "when":
+                    [
+                        {
+                            "method": "GET",
+                            "path": "/tasks/{id};{correlationId}"
+                        },
+                        {
+                            "method": "POST",
+                            "path": "/tasks/{id}"
+                        },
+                        {
+                            "method": "PUT",
+                            "path": "/tasks/{id}"
+                        },
+                        {
+                            "method": "DELETE",
+                            "path": "/tasks/{id}"
+                        }
+                    ],
+                    "with":
+                    {
+                        "capability": "produce",
+                        "topic": "task-command",
```

Comment:
Let's go with plurals for these topic names, such as `task-commands`, where all topics are prefixed with the service name, such as `task`. This will give us a pattern to follow for other examples as well.